### ISSUE TITLE
this version can perform the CIS compliance scan?

### ISSUE THREAD

**Issue by wongcc2012** [elliot-bia/nessus#47]
just a question.
this version can perform the CIS compliance scan?

**Comment by dhruvvyas25** [comment]
No, you can't perform the CIS compliance scan because it does not have compliance plugins installed.

**Comment by elliot-bia** [comment]
I had no working experience of CIS compliance scan , is that means Center for Internet Security ? 
More information will be helpful to solved this, thanks

**Comment by dhruvvyas25** [comment]
Yes, The Center for Internet Security (CIS) developed a series of best practice benchmarks for a variety of applications, operating systems, servers, and databases used within organizations today.

**Comment by elliot-bia** [comment]
Are there any photos can show more details? 
For example：
![image](https://github.com/elliot-bia/nessus/assets/40572216/ae920c07-6b8c-468f-ae2d-1b22ca5e2c1a)

**Comment by dhruvvyas25** [comment]
Yes, that's the CIS scan, but in that Compliance Tab missing in the UI and that's because the nessus doesn't have compliance plugins in /opt/nessus/lib/nessus/plugins/*compliance_check*.

**Comment by wongcc2012** [comment]
Yes you are correct there is the compliance auditing in the official version.
![image](https://github.com/elliot-bia/nessus/assets/111074201/0f31c8e3-ae6d-4e0f-8b5c-f7e6a6899faa)
![image](https://github.com/elliot-bia/nessus/assets/111074201/3dd45f52-6dad-441a-a944-df2ac0bd3d30)

**Comment by dhruvvyas25** [comment]
Can you add the compliance plugins in the cracked nessus ?

**Comment by elliot-bia** [comment]
> Yes, that's the CIS scan, but in that Compliance Tab missing in the UI and that's because the nessus doesn't have compliance plugins in /opt/nessus/lib/nessus/plugins/_compliance_check_.

Sorry, I still don't get it. 
Is this help?
![image](https://github.com/elliot-bia/nessus/assets/40572216/f0600503-0841-40c0-964c-f327f81a593f)

**Comment by elliot-bia** [comment]
Ah, maybe I get what you say. 
Unfortunately， this nessus version base on Nessus-10.x, and I think your nessus version base on something else.
I'm not sure the method I used would be also works the same way

**Comment by wongcc2012** [comment]
this is NOT the complete CIS scan, can you get the nessus EXPERT trial and see what's inside?

**Comment by elliot-bia** [comment]
My daily job doesn't involve this...Nessus-10.x is what I need🤣
I can try from nessus [website](https://www.tenable.com/products/nessus/nessus-expert), but I am not guarantee it will be work

**Comment by wongcc2012** [comment]
![image](https://github.com/elliot-bia/nessus/assets/111074201/601df88b-c6d7-4d19-a3e8-0e13a6f6ecad)
![image](https://github.com/elliot-bia/nessus/assets/111074201/b0a41c7e-368b-45e4-9b49-159917de07ff)
![image](https://github.com/elliot-bia/nessus/assets/111074201/394a7b1a-5356-41b6-9c4b-28e11e149dc5)

I am also using Nessus 10.x EXPERT trial, it has the compliance tab in scanning. then I can choose different compliance / audit to check the target systems.

MY cases: most using the CIS compliance for MS windows systems.

**Comment by jerrycheny** [comment]
Can CIS check be added to this version?

**Comment by Chisj1** [comment]
i was able to use CIS compliance by using a tool to generate a expert activation code and then use that code to download plugin.
